test/test_helper.rb

```diff
@@ -43,9 +43,10 @@ def create_image_blob(filename: "racecar.jpg", content_type: "image/jpeg")
     end
 
     def assert_same_image(fixture_filename, variant)
+      require "mini_magick"
```

Comment:
Turns out, that line is unnecessary. `mini_magik` being in the Gemfile takes care of it, I suppose?